src/EditorFeatures/CSharpTest/Completion/CompletionProviders/Snippets/CSharpElseSnippetCompletionProviderTests.cs

```diff
@@ -239,6 +239,36 @@ public void Method()
             await VerifyCustomCommitProviderAsync(markupBeforeCommit, ItemToCommit, expectedCodeAfterCommit);
         }
 
+        [WpfFact]
+        public async Task InsertElseSnippetSingleLineIfTest()
+        {
+            var markupBeforeCommit =
+@"using System;
+class Program
+{
+    public void Method()
+    {
+        if (true) Console.WriteLine(5);
+        $$
```

Comment:
Should it really be suggested here? I don't think so. Or you just show current behaviour here and it is a subject to change in the future?